docs/content/en/docs/use-cases/process-lifecycle/advanced-process-execution.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Advanced Process execution"
+weight: 2
+icon: "overview"
+description: "Advanced Process Execution monitoring"
+---
+
+## Monitor Elf or Flat binaries execution
+
+Advanced process execution can be performed by using [Tracing Policies](/docs/concepts/tracing-policy) to monitor
+the [execve system call](https://man7.org/linux/man-pages/man2/execve.2.html) path.
+
+If we want to monitor execution of Executable and Linkable Format (ELF) or flat binaries
+before they are actually executed. Then the [process-exec-elf-begin](https://github.com/cilium/tetragon/blob/main/examples/tracingpolicy/process-exec/process-exec-elf-begin.yaml) tracing policy is a good first choice.
+
+{{< note >}}
+Note: the [process-exec-elf-begin](https://raw.githubusercontent.com/cilium/tetragon/main/examples/tracingpolicy/process-exec/process-exec-elf-begin.yaml) tracing policy, will not report the
+different binary format handlers or scripts being executed, but will report
+the final elf or flat binary, like the shebang handler.
+
+To report those another tracing policy can be used.
+{{</ note >}}
+
+Before going forward, verify that all pods are up and running, ensure you
+deploy our Demo Application to explore the Security Observability Events:
+
+```bash
+kubectl create -f https://raw.githubusercontent.com/cilium/cilium/v1.11/examples/minikube/http-sw-app.yaml
+```
+
+It might take several seconds for some pods until they satisfy all the dependencies:
+
+```bash
+kubectl get pods -A
+NAMESPACE            NAME                                         READY   STATUS    RESTARTS   AGE
+default              deathstar-54bb8475cc-6c6lc                   1/1     Running   0          2m54s
+default              deathstar-54bb8475cc-zmfkr                   1/1     Running   0          2m54s
+default              tiefighter                                   1/1     Running   0          2m54s
+default              xwing                                        1/1     Running   0          2m54s
+kube-system          tetragon-sdwv6                               2/2     Running   0          27m
+```
+
+Let's apply the [process-exec-elf-begin](https://raw.githubusercontent.com/cilium/tetragon/main/examples/tracingpolicy/process-exec/process-exec-elf-begin.yaml) Tracing Policy.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/cilium/tetragon/main/examples/tracingpolicy/process-exec/process-exec-elf-begin.yaml
+```
+
+Then start monitoring events with the `tetra` CLI:
+```bash
+kubectl exec -it -n kube-system ds/tetragon -c tetragon -- tetra getevents
+```
+
+In another terminal, `kubectl exec` into the xwing Pod:
+
+```bash
+kubectl exec -it xwing -- /bin/bash
+```
+
+And execute some commands:
+
+```bash
+id
+```
+
+The `tetra` CLI will generate the following [ProcessKprobe](https://tetragon.cilium.io/docs/reference/grpc-api/#processkprobe) events:
```

Comment:
Ah also don't use the complete URLs but use relative path otherwise it will break when we change domain name. Even better use the ref shortcode so that it will detect broken ref at build time.
```suggestion
The `tetra` CLI will generate the following [ProcessKprobe]({{< ref "/docs/reference/grpc-api/#processkprobe" >}}) events:
```

Reply:
done, thanks ;-)